lib/path.js

```diff
@@ -673,7 +673,7 @@ const win32 = {
   },
 
 
-  _makeLong: function _makeLong(path) {
+  toNamespacedPath: function toNamespacedPath(path) {
```

Comment:
This seems to have rather poor type-checking, can we fix that before exposing it publicly?

Reply:
Fix in what way? If the value is not a string it simply returns the value. Given how extensively this method is used by other `fs` methods that perform their own type checking, adding additional type checking to this would be redundant and largely unnecessary.